src/psyclone/tests/generator_test.py

```diff
@@ -129,10 +129,9 @@ def test_script_file_no_extension():
 
 def test_script_file_wrong_extension():
     '''Checks that handle_script() in generator.py raises the excepted
```

Comment:
Not yours but since we're here s/excepted/expected/

Reply:
Done.